lib/source-map-from-file.js

```diff
@@ -50,6 +50,11 @@ function getSourceMapFromFile (filename) {
 }
 
 function dataFromUrl (sourceURL, sourceMappingURL) {
+  const nodeMajorVersion = Number(process.version.slice(1).split('.')[0])
+  /* c8 ignore next 1 */
+  if (nodeMajorVersion === 12) {
+    /* c8 ignore next 14 */
```

Comment:
I'm not sure whether this logic will work, `v8-to-istanbul` just looks at the text of each line, it won't take into account the `if` statement.

Reply:
Removed these lines.